apis/discussionForum/discussionForum.yaml

```diff
@@ -0,0 +1,5451 @@
+          - Backend Route: http://nodebb-service:4567/discussions/api/v2/users/{uid}
+      operationId: UpdateUserInfo
+      parameters:
+      - name: Authorization
+        in: header
+        required: false
+        style: simple
+        explode: false
+        schema:
+          type: string
+      - name: uid
+        in: path
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: integer
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/discussionForumUpdateUserInfo'
+            examples:
+              a:
+                summary: Update user info
+                value:
+                  username: Venkat
+                  email: example@gmail.com
+                  fullname: venkat kadiveti
+                  website: https://example.org/
+                  location: Bangalore
+                  birthday: 02/04/1995
+                  signature: venkat
+        required: false
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumPostOperationResponse'
+        "400":
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-codegen-request-body-name: body
+      x-swagger-router-controller: Users
+    delete:
+      tags:
+      - users
+      summary: Deletes a user from NodeBB
+      description: |-
+        This api is used for deleting user in Nodebb.
+          - The endpoint for **Category** is `/v2/users/{uid}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+          - Backend Route: http://nodebb-service:4567/discussions/api/v2/users/{uid}
+      operationId: DeleteUser
+      parameters:
+      - name: Authorization
+        in: header
+        required: false
+        style: simple
+        explode: false
+        schema:
+          type: string
+      - name: uid
+        in: path
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumPostOperationResponse'
+        "400":
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/discussionForumError'
+      x-swagger-router-controller: Users
+  /discussion/user/v1/create:
+    post:
+      tags:
+      - users
+      summary: Create new user in nodebb OIDC
+      description: |-
+        This api is used for creating the user in Nodebb.
```

Comment:
This description is the same as on line 679. clarify the difference in the two APIs